types/try-catch/tslint.json

```
@@ -0,0 +1,13 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "ban-types": false,
+        "npm-naming": [true, {
+            "mode": "code",
+            "errors":[
+                ["NeedsExportEquals", false]
+            ]
+        }]
+    }
+}
```

```suggestion
    "extends": "dtslint/dt.json"
}
```
why you needed those? You will need inline exclusion for `Function`, which is OK, just add it inline

---

I'm goin to use `try-catch` with `imports` only, so would be great not to add `esModuleInterop` to every `tsconfig.json`. But as I see most packages uses this variant, so I'll need it anyway.